contents/handbook/engineering/posthog-com/small-teams.mdx

```diff
@@ -2,64 +2,100 @@
+2. Team records in our CMS, with most fields editable directly on the small team page
+   - _Team photo, mission, crest_
+3. Small team FAQs
+   - _Aggregated from team member profiles, and from our CMS_
 
 ### Changing the header image
 
 To change your team's image, send it to [#team-website-and-docs](https://posthog.slack.com/archives/C01V9AT7DK4).
 
-### FYIs
+### Changing a mission
+
+Change a small team's mission by clicking _edit mission_ on the small team page.
+
+### Team page content
+
+Any MDX files in the repo will display below the team members and _recently shipped_ sections.
+
+### Quarterly goals
+
+We're moving toward having quarterly goals in their own MDX files, like `2024-Q1.mdx`. This will allow us to show the current team goals, while displaying previous goals in an accordion.
+
+Until then, when adding quarterly goals for a new quarter, add them to the `index.mdx` file and move the previous goals into a section below them (rather than deleting):
+
+```
+## Previous goals
+
+<details>
+<summary>Q3 2024 goals</summary>
 
-- Any MDX files in the repo will display below the team members and _recently shipped_ sections.
+### Goal 1
 
-### In the future
+- Details
+- More details
 
-- We're moving toward having quarterly goals in their own MDX files, like `2024-Q1.mdx`. This will allow us to show the current team goals, while displaying previous goals in an accordion.
-- Moderators will be able to create teams directly on the website (no Strapi required)
+### Goal 2
+
+- Details
+- More details
+
+</details>
+```
+
+See the Website & Docs team's [small team page](/teams/website-docs) as an example of how it will look.
+
+--- 
 
 ## Team management
 
 ### Creating a new small team
 
 #### Initial setup in Strapi
 
-> There's some manual config that needs to be done in our CMS. Ask the Website & Docs team for help here.
+> This config needs to be done in our CMS. Ask the Website & Docs team for help here.
 
 1. Create Team record
 2. Add the [placeholder crest](https://www.figma.com/design/QCYxjm4wVVVvs6UA27xRmz/Website-art?node-id=2043-2&t=aO62tCO12PoIZMt9-1) with the new team's name on it
-3. Go to small team members' profiles, add them to the team, remove from other teams, and assign the team lead
-4. Publish the changes to the team record
 
 #### Setup in the repo
 
 1. Create a directory for the team in `/contents/teams/{team-name}` and duplicate `index.mdx` and `objectives.mdx` from another team as a starting point
 2. Add the team's name to `src/navs/index.js` under the `companyMenu` section in alphabetical order under the _Engineering_ or _Not engineering_ heading
 3. Rebuild the site (after the Strapi configuration) to see the new page
 
+#### Add content to the small team's page
+
+Visit the new small team page to:
+
+1. Add team members
+1. Assign the team lead (click the crown icon)
+1. Add a mission
+1. Ask the Website & Docs team to create a team in Strapi and upload a team photo
```

Comment:
```suggestion
2. Assign the team lead (click the crown icon)
3. Add a mission
4. Ask the Website & Docs team to create a team in Strapi and upload a team photo
```

Reply:
Why though? These get incremented automatically in Markdown!

<img width="695" alt="image" src="https://github.com/user-attachments/assets/027f5c88-3f48-498c-b894-b144e22ae944">

Reply:
Ahh TIL

Reply:
Framing this, because it will never happen again! 😛